.github/workflows/tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Tests
+
+on: [pull_request]
+
+jobs:
+  tests: 
+    name: Tests
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Clone Repository
+        uses: actions/checkout@v3
+
+      - name: Docker Compose
+        run: |
+            docker-compose up -d
+            sleep 5
+
+      - name: Composer Install
+        run: |
+            docker run --rm --volume "$(pwd)":/app composer install --ignore-platform-reqs --optimize-autoloader --no-plugins --no-scripts --prefer-dist
+      
+      - name: Run Tests
+        run: |                
+            docker ps
+            docker-compose exec -T php8 vendor/bin/psalm --show-info=true
+            docker-compose exec -T php8 vendor/bin/phpunit --configuration phpunit.xml
```

Comment:
Instead of calling this way add a script inside the `composer.json` for consistency.

For example take a look at [this](https://github.com/utopia-php/ab/blob/3ca6296f9763da30ce54fd54cfe9d1956797c9a8/composer.json#L13)

Reply:
@2002Bishwajeet Done